TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AprilTagDetectionFrameProcessor.java

```diff
@@ -32,42 +33,46 @@
 import org.opencv.imgproc.Imgproc;
 import org.openftc.apriltag.AprilTagDetection;
 import org.openftc.apriltag.AprilTagDetectorJNI;
-import org.openftc.easyopencv.OpenCvPipeline;
 
 import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A FrameProcessor that detects April tags.
+ *
+ * Adapted with permission from the sample code developed by the OpenFTC Team, 2021.
+ */
+public class AprilTagDetectionFrameProcessor implements WebCam.FrameProcessor {
 
-public class AprilTagDetectionPipeline extends OpenCvPipeline
-{
     private long nativeApriltagPtr;
     private Mat grey = new Mat();
-    private ArrayList<AprilTagDetection> detections = new ArrayList<>();
+    private List<AprilTagDetection> detections = new ArrayList<>();
 
-    private ArrayList<AprilTagDetection> detectionsUpdate = new ArrayList<>();
+    private List<AprilTagDetection> detectionsUpdate = new ArrayList<>();
     private final Object detectionsUpdateSync = new Object();
 
     Mat cameraMatrix;
 
-    Scalar blue = new Scalar(7,197,235,255);
-    Scalar red = new Scalar(255,0,0,255);
-    Scalar green = new Scalar(0,255,0,255);
-    Scalar white = new Scalar(255,255,255,255);
+    private Scalar blue = new Scalar(7, 197, 235, 255);
+    private Scalar red = new Scalar(255, 0, 0, 255);
+    private Scalar green = new Scalar(0, 255, 0, 255);
+    private Scalar white = new Scalar(255, 255, 255, 255);
 
-    double fx;
-    double fy;
-    double cx;
-    double cy;
+    private double fx;
+    private double fy;
+    private double cx;
+    private double cy;
 
     // UNITS ARE METERS
-    double tagsize;
-    double tagsizeX;
-    double tagsizeY;
+    private double tagsize;
+    private double tagsizeX;
+    private double tagsizeY;
```

Comment:
Cleaned up the sample code by using `List` instead of `ArrayList`, and adding `private` where applicable.